src/Avalonia.OpenGL/Angle/AngleEglInterface.cs

```diff
@@ -7,8 +7,8 @@ namespace Avalonia.OpenGL.Angle
 {
     public class AngleEglInterface : EglInterface
     {
-        [DllImport("libegl.dll", CharSet = CharSet.Ansi)]
-        static extern IntPtr eglGetProcAddress(string proc);
+        [DllImport("libglesv2.dll", CharSet = CharSet.Ansi)]
```

Comment:
I'd rather not do that, libegl.dll is an integral part of ANGLE.

Reply:
must be some build issues?